autopilot/store.go

```diff
@@ -0,0 +1,117 @@
+package autopilot
```

Comment:
Can this store no longer be `internal`?

Reply:
Hm now that we have the `api` package we can move it back indeed. Originally I couldn't do that because I needed to import the autopilot `Config`. I still feel the store could live in the `autopilot` package because it's the only store that's not on the `bus` but I'll move it.